File: 1972a/main.py
def I():
    return input()


def II():
    return int(input())


def MII():
    return map(int, input().split())


def LI():
    return list(input().split())


def LII():
    return list(map(int, input().split()))

a = II()
for _ in range(a):
    b = II()
    r = l = 0

    L = LII()
    M = LII()
    ans = 0
    while r < b and l < b:
        while r < b and L[l] > M[r]:
            ans += 1
            r += 1
        l += 1
        r += 1
    print(ans )

File: 1972a/main.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func solve(reader *bufio.Reader) {
	var b int
	fmt.Fscan(reader, &b)
	var s string
	fmt.Fscan(reader, &s)
	ans := 0
	for _, x := range s {
		if x == 'U' {
			ans += 1

		}
	}
	if ans&1 == 1 {
		fmt.Println("YES")
	} else {
		fmt.Println("NO")
	}
}
func main() {
	reader := bufio.NewReader(os.Stdin)
	var a int
	fmt.Fscan(reader, &a)
	for i := 1; i <= a; i++ {
		solve(reader)
	}
}
